sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -71,7 +71,9 @@ object CurrentOrigin {
   }
 }
 
+// scalastyle:off
```

Comment:
I actually tried this by assigning a custom name to the rule and it ended up breaking the Scalastyle error message output for this rule. If that had not happened, I would have used a more specific exclude here.